apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -59,79 +75,143 @@
+                    </button>
+                  </div>
+                </div>
               </div>
             </div>
-            <div class="expand filter">
-              <div class="filter-wrap">
-                <input
-                  class="form-control"
-                  type="text"
-                  v-model="filter.value"
-                  :placeholder="builderPlaceholder"
-                  ref="valueInput"
-                >
+            <div class="right-section">
+              <div class="ghost-add-reply">
+                <div class="btn-wrap">
+                  <button class="btn btn-flat btn-fab">
+                    <i class="material-icons">add</i>
+                  </button>
+                </div>
+                <div class="btn-wrap">
+                  <button class="btn btn-primary btn-fab">
+                    <i class="material-icons">search</i>
+                  </button>
+                </div>
+              </div>
+              <div class="filter-add-apply">
+                <div class="row fixed">
+                  <div class="btn-wrap add-filter">
+                    <button
+                      class="btn btn-flat btn-fab"
+                      type="button"
+                      title="Add filter"
+                      @click="addFilter"
+                    >
+                      <i class="material-icons">add</i>
+                    </button>
+                  </div>
+                  <!-- TODO 2 : when you add a filter, it should scroll automatically -->
+                  <div class="btn-wrap" ref="filterButtonWrapper">
+                    <button
+                      class="btn btn-primary btn-fab"
+                      type="submit"
+                      title="Apply filter"
+                    >
+                      <i class="material-icons">search</i>
+                    </button>
+                  </div>
+                </div>
+              </div>
+              <div class="btn-wrap" v-for="(filter, index) in additionalFilters" :key="index">
                 <button
+                  class="btn btn-flat btn-fab remove-filter"
                   type="button"
-                  class="clear btn-link"
-                  @click.prevent="filter.value = ''"
+                  title="Remove filter"
+                  @click="removeFilter(index)"
                 >
-                  <i class="material-icons">cancel</i>
+                  <i class="material-icons">remove</i>
                 </button>
               </div>
             </div>
-            <div class="btn-wrap">
-              <button
-                class="btn btn-primary btn-fab"
-                type="submit"
-                title="Filter"
-              >
-                <i class="material-icons">search</i>
-              </button>
-            </div>
           </div>
         </form>
+        <div class="filter-drag-icon"><i class="material-icons">drag_handle</i></div>
```

Comment:
You know what I think it's great without the resizing. Can we just remove the resizing? I think even adding 5-6 filters there is still loads of space

Reply:
Sure, I'll do that!